modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase

```diff
@@ -232,33 +232,6 @@ definition {
 			fieldGroupLabelList = "Copy of Text,Copy of Numeric,Copy of Date");
 	}
 
-		WebContentStructures.saveCP();
```

Comment:
Hey @victortrajano , next time when you need to alter a commit with a fix you can use amend git command, in order to not create a commit like this one. Thanks